tools/src/commands/AddChangelog.ts

```diff
@@ -26,17 +26,25 @@ async function checkOrAskForOptions(options: ActionOptions): Promise<ActionOptio
     filter: (s: string) => s.trim(),
     validate: lengthValidator,
   };
+  const listOfStringsValidator = {
```

Comment:
Since it's not used anywhere else, I feel that it would be more clear if it's used directly instead of being spread in the question. I'm fine with both though, just let me know if you want to change it.